components/lang/src/main/java/io/cellery/CelleryUtils.java

```diff
@@ -112,8 +115,36 @@ public static void writeToFile(String context, String targetPath) throws IOExcep
             writer.close(); //don't add this to finally, because the text will not be flushed
             return stringWriter.toString();
         } catch (IOException e) {
-            throw new BallerinaException("Error occurred while generating yaml definition.");
+            throw new BallerinaException("Error occurred while generating yaml definition." + e.getMessage());
         }
     }
 
+    /**
+     * Copy file target/resources directory.
+     *
+     * @param sourcePath source file/directory path
+     */
+    public static void copyResourceToTarget(String sourcePath) {
+        File src = new File(sourcePath);
+        String targetPath = TARGET + File.separator + RESOURCES + File.separator + src.getName();
+        File dst = new File(targetPath);
+        // if source is file
+        try {
+            if (Files.isRegularFile(Paths.get(sourcePath))) {
+                if (Files.isDirectory(dst.toPath())) {
+                    // if destination is directory
+                    FileUtils.copyFileToDirectory(src, dst);
+                } else {
+                    // if destination is file
+                    FileUtils.copyFile(src, dst);
+                }
+            } else if (Files.isDirectory(Paths.get(sourcePath))) {
+                FileUtils.copyDirectory(src, dst);
+            }
+
+        } catch (IOException e) {
+            throw new BallerinaException("Error occured while copying resource file " + sourcePath +
+                    ". " + e.getMessage());
```

Comment:
Using e.getMessage() will drop the complete stack trace.

Reply:
As per the chat with AnuruddhaL, this should be logged to ballerina-internal.log file.